controller/RestTest.php

```diff
@@ -15,78 +15,124 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
- *
+ * Copyright (c) 2017-2020 (original work) Open Assessment Technologies SA;
  */
 
+declare(strict_types=1);
+
 namespace oat\taoDeliveryRdf\controller;
 
+use common_report_Report as Report;
+use tao_helpers_Http as HttpHelper;
+use common_exception_Error as Error;
 use oat\tao\model\import\ImporterNotFound;
 use oat\tao\model\taskQueue\TaskLogInterface;
+use tao_actions_RestController as RestController;
+use common_exception_NotFound as NotFoundException;
 use oat\taoDeliveryRdf\model\tasks\ImportAndCompile;
+use common_exception_BadRequest as BadRequestException;
+use common_exception_NotImplemented as NotImplementedException;
+use common_exception_MissingParameter as MissingParameterException;
 
 /**
  * Class RestTest
+ *
  * @package oat\taoDeliveryRdf\controller
+ *
  * @author Aleh Hutnikau, <hutnikau@1pt.com>
  */
-class RestTest extends \tao_actions_RestController
+class RestTest extends RestController
 {
-    const REST_IMPORTER_ID = 'importerId';
-    const REST_FILE_NAME = 'testPackage';
-    const REST_DELIVERY_PARAMS = 'delivery-params';
-    const REST_DELIVERY_CLASS_LABEL = 'delivery-class-label';
+    public const REST_IMPORTER_ID = 'importerId';
+    public const REST_FILE_NAME = 'testPackage';
+    public const REST_DELIVERY_PARAMS = 'delivery-params';
+    public const REST_DELIVERY_CLASS_LABELS = 'delivery-class-labels';
+
+    /** @deprecated Use \oat\taoDeliveryRdf\controller\RestTest::REST_DELIVERY_CLASS_LABELS instead */
```

Comment:
Currently, the parameter's support looks abandoned, not deprecated.

Please consider still accepting and using its value when mapping the data to `ImportAndCompile::createTask()`.

Reply:
Added a backward compatibility [here](https://github.com/oat-sa/extension-tao-delivery-rdf/pull/360/commits/7aa816e28bcd35b9a3051b770335e72ae6f2e39b).